.github/workflows/swagger-bump.yml

```diff
@@ -0,0 +1,20 @@
+name: swagger-bump
+on:
+  # Workaround for bumping the swagger.yml file on release-please PRs
+  push:
+    branches:
+      - release-*
+jobs:
+  swagger-bump:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm install
```

Comment:
Should we use `npm ci` here?

Reply:
I thought `npm ci` only installed `dependencies` and not `devDependencies` (which we need here) but now I'm uncertain - https://docs.npmjs.com/cli/v6/commands/npm-ci - and according to the docs doesn't seem like it. I'll change it, thanks @erezrokah 👍